cf-custom-resources/lib/env-controller.js

```diff
@@ -203,10 +206,9 @@ exports.handler = async function (event, context) {
         responseData = await Promise.race([
           exports.deadlineExpired(),
           controlEnv(
-            "Delete",
             props.EnvStack,
             props.Workload,
-            props.Parameters
+            []
```

Comment:
nit: can we add a comment here as it might not be obvious why
```suggestion
            [] // Set to empty to denote that Workload should not be included in any env stack parameter.
```

Reply:
Wow brilliant insight Efe, done!